ironfish-rust/src/transaction/mod.rs

```diff
@@ -80,8 +75,6 @@ pub struct ProposedTransaction {
 impl ProposedTransaction {
     pub fn new(spender_key: SaplingKey) -> ProposedTransaction {
         ProposedTransaction {
-            binding_signature_key: <jubjub::Fr as Field>::zero(),
```

Comment:
We're no longer keeping track of these throughout the entire life-cycle. Instead we calculate them during the build phase. This removes the ability to `check_value_consistency` at each point of adding spends/outputs, but it's not something we were really doing anyway.